scripts/calculate_metrics.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import json
+import os
+
+import yaml
+from functions import opts2dmpairArgs
+
+from modsim2.data.loader import DMPair
+
+
+def main(
+    dataset_config: dict,
+    metric_config: dict,
+    experiment_group: str,
+    dataset_index: int,
+    seed_index: int,
+):
+    experiment_pair_name = f"{experiment_group}_{dataset_index}_{seed_index}"
+    dmpair_kwargs = opts2dmpairArgs(
+        opt=dataset_config["experiment_groups"][experiment_group][dataset_index],
+        seed=dataset_config["seeds"][seed_index],
+    )
+
+    dmpair = DMPair(**dmpair_kwargs, metric_config=metric_config)
+    metrics = {
+        "experiment_pair_name": experiment_pair_name,
+        **dmpair.compute_similarity(),
+    }
+
+    # JSON file path
+    root_path = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+    data_path = os.path.join(root_path, "data")
+    out_file_path = os.path.join(data_path, "metrics.json")
```

Comment:
either make this path configurable via argparse (with default) or clearly specify in README where this is going to land and log telling the user you've written there. I'd also use `"results"` or `"output"` rather than `"data"` here.

Reply:
Have added to README, will add the log shortly

Reply:
Have now added the log